addons/sale/models/sale_order.py

```diff
@@ -1496,6 +1485,21 @@ def _prepare_down_payment_section_line(self, **optional_values):
         del context
         return down_payments_section_line
 
+    def _get_amounts(self, order_lines):
+        self.ensure_one()
+        if self.company_id.tax_calculation_rounding_method == 'round_globally':
+            tax_results = self.env['account.tax']._compute_taxes([
+                line._convert_to_tax_base_line_dict()
+                for line in order_lines
+            ])
+            totals = tax_results['totals']
+            amount_untaxed = totals.get(self.currency_id, {}).get('amount_untaxed', 0.0)
+            amount_tax = totals.get(self.currency_id, {}).get('amount_tax', 0.0)
+        else:
+            amount_untaxed = sum(order_lines.mapped('price_subtotal'))
+            amount_tax = sum(order_lines.mapped('price_tax'))
+        return amount_untaxed, amount_tax
```

Comment:
Let's keep disregarding the global discount in the amounts comparison, as it was before.
We'll fix this if it comes back in another ticket.

Reply:
Done.